Microsoft.Toolkit.Uwp.SampleApp/App.xaml

```diff
@@ -125,12 +131,144 @@
+            <Setter Property="Template">
+                <Setter.Value>
+                    <ControlTemplate TargetType="controls1:HamburgerMenu">
+                        <Grid>
+                            <SplitView x:Name="MainSplitView"
+                                   CompactPaneLength="{TemplateBinding CompactPaneLength}"
+                                   DisplayMode="{TemplateBinding DisplayMode}"
+                                   IsPaneOpen="{Binding IsPaneOpen, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}"
+                                   IsTabStop="False"
+                                   OpenPaneLength="{TemplateBinding OpenPaneLength}"
+                                   PaneBackground="{TemplateBinding PaneBackground}"
+                                   PanePlacement="{TemplateBinding PanePlacement}">
+                                <SplitView.Pane>
+                                    <Grid x:Name="PaneGrid">
+                                        <Grid.RowDefinitions>
+                                            <RowDefinition Height="Auto" />
+                                            <RowDefinition Height="Auto" />
+                                            <RowDefinition />
+                                            <RowDefinition Height="Auto" />
+                                        </Grid.RowDefinitions>
+                                        <Grid Grid.Row="0"
+                                              Height="{TemplateBinding HamburgerHeight}" />
+                                        <Grid Grid.Row="1"
+                                              Height="{TemplateBinding HamburgerHeight}">
+                                            <Button Name="SearchButton"
+                                                Width="{TemplateBinding HamburgerWidth}"
+                                                Height="{TemplateBinding HamburgerHeight}"
+                                                Padding="0"
+                                                VerticalAlignment="Top"
+                                                AutomationProperties.Name="Search button"
+                                                Background="Transparent"
+                                                BorderThickness="0"
+                                                Visibility="{Binding ElementName=MainSplitView, Path=IsPaneOpen, Converter={StaticResource BoolToVisibilityConverter}, ConverterParameter=True}"
+                                                TabIndex="1">
+                                                <SymbolIcon Symbol="Find"
+                                                            Foreground="{TemplateBinding Foreground}"/>
+                                            </Button>
+                                            <SearchBox x:Name="SearchBox" 
+                                                       Margin="5"
```

Comment:
FYI we prefer nobody use SearchBox anymore and use AutoSuggestBox as the replacement.